<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Archer's Stock Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="A_LCA_page" native="true">
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>150</y>
          <width>125</width>
          <height>66</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="goto_login_button">
           <property name="text">
            <string>Login</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="goto_CA_button">
           <property name="text">
            <string>Create Account</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="B_login_page" native="true">
       <widget class="QPushButton" name="login_page_button">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>220</y>
          <width>310</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
       <widget class="QPushButton" name="login_page_button_back">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>270</y>
          <width>310</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="login_page_login_input">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>120</y>
          <width>311</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Username</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="login_page_pw_input">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>170</y>
          <width>311</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Password</string>
        </property>
       </widget>
       <widget class="QLabel" name="login_notice_label">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>69</y>
          <width>311</width>
          <height>31</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="C_CA_page" native="true">
       <widget class="QPushButton" name="CA_page_button">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>230</y>
          <width>310</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Create Account</string>
        </property>
       </widget>
       <widget class="QPushButton" name="CA_page_button_back">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>280</y>
          <width>310</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="CA_page_pw_input">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>170</y>
          <width>311</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Password</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="CA_page_login_input">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>120</y>
          <width>311</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Username</string>
        </property>
       </widget>
       <widget class="QLabel" name="CA_notice_label">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>70</y>
          <width>311</width>
          <height>31</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="D_welcome_page">
       <widget class="QLabel" name="welcome_page_welcome_label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>801</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:700;&quot;&gt;Welcome, John Doe!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
       <widget class="QLabel" name="welcome_page_started_label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>100</y>
          <width>801</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#0000d5;&quot;&gt;Let's get started.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="welcome_view_portfolio">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>210</y>
          <width>191</width>
          <height>141</height>
         </rect>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">                background-color: #800080;
                color: white;
                padding: 10px;
                border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>View Your Portfolio</string>
        </property>
       </widget>
       <widget class="QPushButton" name="welcome_buy_sell">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>210</y>
          <width>191</width>
          <height>141</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">                background-color: #4CAF50;
                color: white;
                padding: 10px;
                border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>Buy/Sell Stocks</string>
        </property>
       </widget>
       <widget class="QPushButton" name="welcome_gen_reports">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>210</y>
          <width>191</width>
          <height>141</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">                background-color: #000080;
                color: white;
                padding: 10px;
                border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>Generate Reports</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="E_buy_sell_page">
       <widget class="QWidget" name="bs_left_bar" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>91</width>
          <height>411</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #00008B</string>
        </property>
        <widget class="QPushButton" name="bs_portfolio">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>71</width>
           <height>61</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #FFFFFF;
padding: 5px;
border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>Portfolio</string>
         </property>
        </widget>
        <widget class="QPushButton" name="bs_bs">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>71</width>
           <height>61</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #FFFFFF;
padding: 5px;
border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>Buy/Sell</string>
         </property>
        </widget>
        <widget class="QPushButton" name="bs_reports">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>71</width>
           <height>61</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #FFFFFF;
padding: 5px;
border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>Reports</string>
         </property>
        </widget>
       </widget>
       <widget class="QLineEdit" name="bs_ticker_input">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>170</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-weight: bold;
color: black;</string>
        </property>
        <property name="placeholderText">
         <string>Ticker Symbol</string>
        </property>
       </widget>
       <widget class="QWidget" name="bs_ticker_display" native="true">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>30</y>
          <width>411</width>
          <height>361</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid black;</string>
        </property>
        <widget class="QLabel" name="bs_info_ticker">
         <property name="geometry">
          <rect>
           <x>87</x>
           <y>9</y>
           <width>251</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 0px;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:700;&quot;&gt;Ticker&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
        </widget>
        <widget class="QLabel" name="bs_info_name">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>40</y>
           <width>251</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 0px;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Name of Company&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
        </widget>
        <widget class="QLabel" name="bs_info_price">
         <property name="geometry">
          <rect>
           <x>87</x>
           <y>100</y>
           <width>251</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 0px;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:700;&quot;&gt;$0.00&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
        </widget>
        <widget class="QLabel" name="bs_info_details">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>170</y>
           <width>371</width>
           <height>161</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 0px;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Exchange:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Sector:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;52 Week High:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;52 Week Low: &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
        </widget>
        <widget class="QStackedWidget" name="bs_stacked_widget">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>290</y>
           <width>361</width>
           <height>61</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 0px;</string>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="bs_buy_sell_main">
          <widget class="QPushButton" name="bs_buy_button">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>161</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid black;</string>
           </property>
           <property name="text">
            <string>Buy</string>
           </property>
          </widget>
          <widget class="QPushButton" name="bs_sell_button">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>10</y>
             <width>161</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid black;</string>
           </property>
           <property name="text">
            <string>Sell</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="bs_buy_page">
          <widget class="QLineEdit" name="bs_buy_quantity">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>10</y>
             <width>171</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid black;
font-weight: bold;
color: black;</string>
           </property>
           <property name="placeholderText">
            <string>Enter Quantity</string>
           </property>
          </widget>
          <widget class="QPushButton" name="bs_buy_confirm">
           <property name="geometry">
            <rect>
             <x>300</x>
             <y>10</y>
             <width>61</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid black;</string>
           </property>
           <property name="text">
            <string>Confirm</string>
           </property>
          </widget>
          <widget class="QPushButton" name="bs_buy_cancel">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>101</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid black;</string>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="bs_sell_page">
          <widget class="QLineEdit" name="bs_sell_quantity">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>20</y>
             <width>171</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid black;
font-weight: bold;
color: black;</string>
           </property>
           <property name="placeholderText">
            <string>Enter Quantity</string>
           </property>
          </widget>
          <widget class="QPushButton" name="bs_sell_confirm">
           <property name="geometry">
            <rect>
             <x>290</x>
             <y>20</y>
             <width>61</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid black;</string>
           </property>
           <property name="text">
            <string>Confirm</string>
           </property>
          </widget>
          <widget class="QPushButton" name="bs_sell_cancel">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>20</y>
             <width>101</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid black;</string>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
          <widget class="QLabel" name="bs_sell_quantity_label">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>351</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>You currently have X shares of Y in your portfolio.</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="bs_order_confirmed">
          <widget class="QLabel" name="bs_processing_label">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>331</width>
             <height>41</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:700;&quot;&gt;Processing Order&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
       <widget class="QPushButton" name="bs_submit_ticker">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>210</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
       <widget class="QLabel" name="bs_info_1">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>130</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Enter a ticker symbol below!</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="F_portfolio_page"/>
      <widget class="QWidget" name="G_reports_page"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>899</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
